base/deprecated.jl

```diff
@@ -1595,6 +1595,8 @@ end
 # issue #6466
 # `write` on non-isbits arrays is deprecated in io.jl.
 
+@deprecate isconcrete isconcrete
```

Comment:
fixed in 38134218d37579ae4623051832a55feab5d1092a - I think this was a mistake during the conflict resolution in fd4283c865beae41bbadafe499dcf9d05a8b0a93